torax/_src/neoclassical/transport/tests/angioni_sauter_test.py

```diff
@@ -116,17 +112,17 @@ def test_angioni_sauter_against_reference_values(self):
 ])
 
 _EXPECTED_CHI_NEO_E = np.array([
-    -0.0,
-    -0.00203451,
-    -0.00298301,
-    -0.00379278,
-    -0.0045157,
-    -0.00518874,
-    -0.00587365,
-    -0.00663125,
-    -0.00753242,
-    -0.00873426,
-    -0.00994788,
+    0.0,
+    0.0,
+    0.0,
+    0.0,
+    0.0,
+    0.0,
+    0.0,
+    0.0,
+    0.0,
+    0.0,
+    0.0,
```

Comment:
My one concern is that this is no longer testing the chi_neo_e implementation sufficiently, as it is now clipped to 0.

Reply:
Can you change the clipping here and then have another test where the clipping is checked?